test/inductor/test_b2b_gemm.py

```diff
@@ -0,0 +1,28 @@
+import torch
+
+from torch._inductor.test_case import run_tests, TestCase
+
+class B2BGEMMTest(TestCase):
+
+    @torch._inductor.config.patch(b2b_gemm_pass=True)
+    def test_b2b_gemm_1(self):
+        """ sizes match the supported (hardcoded) value """
+        f = lambda m1, m2, m3: torch.matmul(torch.matmul(m1, m2), m3)
+        f_opt = torch.compile(f)
+        A = torch.randn((512, 32), device="cuda", dtype=torch.float16)
+        B = torch.randn((32, 256), device="cuda", dtype=torch.float16)
+        C = torch.randn((256, 32), device="cuda", dtype=torch.float16)
```

Comment:
Just as important as comparing the output is to make sure the b2b gemm was actually applied. I can think of a couple ways to do it; you might have better ideas:
1) Use unit test mock object with the "wraps=" argument to wrap an internal method. Then you can assert that the mock was called.
2) Some of our existing tests use an internal `run_and_get_code()` function to actually get the generated code as a string (just grep for it in the tests/ subdir). Then you can assert on some substring that you expect only when your b2b gemm is applied.

I guess 2 strikes me as a bit more consistent with our other tests, but like I said, just a suggestion; you may have other ideas.

Reply:
Now uses `run_and_get_code()`.